src/content/docs/blog/2025/06/30/mid-year-update/index.md

```diff
@@ -0,0 +1,185 @@
+
+Towards the end of May, we put the first prototype infra to the test, and then iteratively fixed bugs and built out missing functionality to the point of being able to put our Minimum Viable Product (MVP) into production on our build infrastructure.
+
+This MVP will serve as the development base of the code that will be used for all future package builds.
+
+<details>
+<summary>What do we mean by infra?</summary>
+
+Our [infra](https://github.com/AerynOS/infra) is comprised of the Summit, Avalanche and Vessel service components:
+
+- **Summit:** Package build-controller, build-orchestrator and build-dashboard, monitors recipes tree and automatically builds new, incoming recipes once they show up
+- **Avalanche:** build agent middleware, takes build orders from summit and builds them with boulder on a remote system, sends build logs back in real time to summit, and reports the build result to summit at the end of the build.
+- **Vessel:** Package repository manager. Summit tells Vessel which packages and other build artefacts to expect from a build task that Avalanche has completed, and then Avalanche pushes those packages and build artefacts to Vessel, which then saves them in the appropriate place and re-indexes the repository with the new packages, so users can install/update them.
+
+</details>
+
+
+We have some cool features planned in AerynOS that we envision will make package maintenance a lot easier to manage through smart use of automation.
+
+Until these features are implemented, however, maintaining the AerynOS repository will remain somewhat human resource intensive. This is the main reason why the repository is consciously being kept "small" for now, with us deliberately focusing on having packages that will help developers and contributors to improve AerynOS, while still delivering a nice Daily Driver experience.
+
+Until the new features are implemented, this will necessarily be a balancing act between maintaining the package repository so it doesn't go stale vs. having the development time to implement the new features.
+
+### How did you test it?
+
+Aside from porting the infra code to Rust, proper testing was required to yield confidence that packages were both successfully built on the new infra and that they worked as expected. 
+
+The end goal was to prove that we were able to rebuild the full AerynOS recipes repository (currently at ~950 recipes) from start to finish without infra-related build errors on the new infra.
+
+To enable the rebuild, ermo set up a distributed build cluster of four builders of varying hardware specifications. A separate branch of the 'recipes' repository was created, and was used to test both the Rust infra and to land packages for internal testing without them being seeded to user installs.
```

Comment:
First time 'recipes' is quoted like this. I'd either do it throughout the whole post or nowhere.